framework/include/controls/Control.h

```diff
@@ -114,11 +114,11 @@ class Control :
   void setControllableValueByName(const std::string & tag, const std::string & param_name, const T & value, bool warn_when_values_differ = false);
   ///@}
 
-private:
-
   /// A reference to the InputParameterWarehouse which is used for access the parameter objects
   InputParameterWarehouse & _input_parameter_warehouse;
 
+private:
+
   /// Helper method for retrieving controllable parameters
   template<typename T>
   ControllableParameter<T> getControllableParameterHelper(const MooseObjectParameterName & desired, bool warn_when_values_differ, bool mark_as_set = false);
```

Comment:
I don't want this to be available to users, this is simply to reduce code duplication for the other methods that they are supposed to call.